tests/core-chatbot/main.go

```
@@ -0,0 +1,936 @@
+			content = *msg.Content.ContentStr
+		} else if msg.Content.ContentBlocks != nil {
+			var textParts []string
+			for _, block := range *msg.Content.ContentBlocks {
+				if block.Text != nil {
+					textParts = append(textParts, *block.Text)
+				}
+			}
+			content = strings.Join(textParts, "\n")
+		}
+
+		role := strings.Title(string(msg.Role))
```

golangci-lint (1.64.8)

SA1019: strings.Title has been deprecated since Go 1.18 and an alternative has been available since Go 1.0: The rule Title uses for word boundaries does not handle Unicode punctuation properly. Use golang.org/x/text/cases instead. 

(staticcheck)